src/pages.css

```diff
@@ -0,0 +1,388 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700;800&family=Poiret+One&family=Roboto+Slab:wght@200;300;400;500;600;700&family=Shadows+Into+Light&display=swap');
+
+.flex {
+  display: flex;
+}
+
+.ff-2 {
+  font-family: 'Roboto Slab', serif;
+}
+
+.fs-1 {
+  font-size: 11pt;
+}
+
+.w100 {
+  width: 100%;
+}
+
+header {
+  position: relative;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  padding: 0 5%;
+}
+
+header h1 {
+  color: #0290ff;
+  font-size: 1rem;
+}
```

Comment:
Overall, you have done a beautiful job in styling this project according to the provided design. You just need to make a few changes to make perfect.

| Project Design | Zeplin Design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/156847775-b5490a9d-7d6f-4cd9-9fdb-2c9f28e8f04d.png) | ![bookstore](https://user-images.githubusercontent.com/4247740/156848007-3aa8ec93-bf2f-4211-8e17-aac7b7ffe788.png) |

- [ ] On comparing both designs, I think the shadow on the book component is unnecessary and can be removed
- [ ] Also the  form background is not meant to be white as seen in the original design.